CHANGELOG.md

```diff
@@ -40,6 +40,28 @@ librdkafka v2.1.0 is a feature release:
  * When using `rd_kafka_seek_partitions`, the remaining timeout was
    converted from microseconds to milliseconds but the expected unit
    for that parameter is microseconds.
+ * Fixed known issues related to Batch Consume APIs mentioned in v2.0.0
+   release notes.
+ * Fixed `rd_kafka_consume_batch()` and `rd_kafka_consume_batch_queue()`
+   intermittently updating `app_offset` and `store_offset` incorrectly when
+   **pause** or **resume** was being used for a partition.
+ * Fixed `rd_kafka_consume_batch()` and `rd_kafka_consume_batch_queue()`
+   intermittently skipping offsets when **pause** or **resume** was being
+   used for a partition.
+
+
+## Known Issues
+
+### Consume Batch API
+
+ * When `rd_kafka_consume_batch()` and `rd_kafka_consume_batch_queue()` APIs are used with
+   any of the **seek**, **pause**, **resume** or **rebalancing** operation, `on_consume`
+   iterceptors might be called incorrectly (maybe multiple times) for not consumed message.
+ * Using multiple instances of `rd_kafka_consume_batch()` and `rd_kafka_consume_batch_queue()`
+   APIs in conjunction with any of the **seek**, **pause**, **resume** or **rebalancing**
+   operation concurrently is not thread safe and will result in undefined behaviour. This usecase
+   is not supported and might not be supported in future as well. Use multiple instances of
+   librdkafka with different partitions to achieve the same.
```

Comment:
Maybe add "or use multiple threads to process the batch read by a single thread"

Reply:
Moved to Introduction.md